roles/cloud-cloudstack/tasks/prompts.yml

```diff
@@ -0,0 +1,7 @@
+---
+- block:
+  - pause:
+      prompt: |
+        What region should the server be located in?
```

Comment:
Can we put existing regions into this prompt? As in the other roles. Not sure if there is a modules for ansible already, but CloudStack API has a method for that https://cloudstack.apache.org/api/apidocs-4.9/apis/listRegions.html

Reply:
Good point! I used the `cs` binary install alongside `cs` lib in the venv since no current `cs_*` ansible module allows to perform this for the moment.

I had to update `requirements.txt` to include `jmespath` in order to call `json_query` to parse output.

Reply:
We are trying to avoid updating `requirements.txt`. Jinja2 filters can handle json pretty well.

Reply:
`json_query` is part of jinja2 no? What would be the other method you recommend?

Reply:
For example you may take a loot at how it's implemented [here](https://github.com/trailofbits/algo/blob/master/roles/cloud-lightsail/tasks/prompts.yml)

Reply:
thanks, fixed

Reply:
OK, thanks. We also try to not use any external clis from the playbooks. 

> shell "{{ cloudstack_venv }}/bin/cs" listZones

Could you, please, eliminate this one for now, and just add a default variable with the regions (see [this](https://github.com/trailofbits/algo/blob/master/roles/cloud-azure/defaults/main.yml#L3) for example).
Ideally we need to create an external module to discover the regions (eg: [lightsail](https://github.com/trailofbits/algo/blob/master/library/lightsail_region_facts.py)). I can do it later, after we merge this PR, or you can cover it here

Reply:
done, I've also opened an [issue](https://github.com/ansible/ansible/issues/53916) and will wait feedbacks from the module's maintainer on how to proceed.